includes/filerepo/backend/SwiftFileBackend.php

```diff
@@ -571,7 +573,15 @@ protected function addMissingMetadata( CF_Object $obj, $path ) {
 			return true; //nothing to do
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS139)